### PR TITLE
Update package.json and README.md to publish it onto npm

### PR DESCRIPTION
**Pull request by somniumism**

### CODE REVIEW

**Comment by joojis** [comment]
# [Codecov](https://codecov.io/gh/CodeChain-io/foundry-primitives-js/pull/2?src=pr&el=h1) Report
> Merging [#2](https://codecov.io/gh/CodeChain-io/foundry-primitives-js/pull/2?src=pr&el=desc) into [master](https://codecov.io/gh/CodeChain-io/foundry-primitives-js/commit/1d30b6d97964cd11a87ab4d020d92dd397ffb4fd&el=desc) will **not change** coverage by `%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/CodeChain-io/foundry-primitives-js/pull/2?src=pr&el=tree)

```diff
@@           Coverage Diff           @@
##           master       #2   +/-   ##
=======================================
  Coverage   91.75%   91.75%           
=======================================
  Files          16       16           
  Lines         983      983           
  Branches      216      216           
=======================================
  Hits          902      902           
  Misses         81       81           
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/CodeChain-io/foundry-primitives-js/pull/2?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/CodeChain-io/foundry-primitives-js/pull/2?src=pr&el=footer). Last update [1d30b6d...a0a5884](https://codecov.io/gh/CodeChain-io/foundry-primitives-js/pull/2?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by somniumism** [comment]
@HoOngEe Could you check it first?

**Comment by HoOngEe** [comment]
@somniumism LGTM, but It can be merged after the #1 is merged

**Review by HoOngEe — APPROVED** [review]

**Pull request merged**
